src/Command/ProcessImageCommand.php

```
@@ -0,0 +1,96 @@
+        $io = new SymfonyStyle($input, $output);
+        $imagePath = $input->getArgument('image');
+
+        if (!is_file($imagePath)) {
+            $io->error('Image file not found: ' . $imagePath);
+            return Command::FAILURE;
+        }
+
+        $endpoint = rtrim($this->embedHost, '/') . ':' . $this->embedPort . '/image-embedding';
+
+        try {
+            $dataPart = new DataPart(file_get_contents($imagePath), basename($imagePath));
```

PHPStan (2.1.15)

Instantiated class Symfony\Component\Mime\Part\DataPart not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)